javatests/arcs/sdk/jvm/CollectionImplTest.kt

```diff
@@ -31,11 +32,13 @@ class CollectionImplTest {
     private val HANDLE_NAME = "HANDLE_NAME"
     private val DUMMY_VALUE1 = DummyEntity("111")
     private val DUMMY_VALUE2 = DummyEntity("222")
+    @Mock private lateinit var action: (Set<DummyEntity>) -> Unit
```

Comment:
I personally prefer testing without mocks, but I will not force you to change if you like this.

Reply:
Just to leave a dissenting opinion: this is an ideal time to use a mock. You want to check how that a callback was invoked, and how many times. Mockito has a great API for this :-)